src/App.css

```diff
@@ -53,6 +53,96 @@ section {
   color: #888;
 }
 
+.home-header {
```

Comment:
this is ok, but since we are using nextui and tailwind, it would be great to use them. Also, it is more convenient, responsive, and handle all tedious stuff. Look at their documentation: https://nextui.org/docs/components/button

Reply:
But don't work on this on this pr. This should be done in a different branch and pr